### PR TITLE
Updated c8ycli documentation.

### PR DESCRIPTION
**Pull request by eniosultan**

### CODE REVIEW

File: content/web/development-tools-bundle/c8y-cli.md [hunk 1]
```diff
@@ -51,6 +51,57 @@ $ c8ycli new cockpit cockpit
...
+> 1011.0.X
+> 1009.0.X
+> 1007.0.X
+> 1006.0.X
+> other
+```
+
+In the second step, the base scaffolding version must be selected. The interface will provide the last 5 major GA releases, the `latest` and `next` versions as options. Additionally a version can be manually entered by selecting the `other` option.
+
+**Step 2(other):**
```
Comment (rssag) [inline content/web/development-tools-bundle/c8y-cli.md:79]:
```suggestion
**Step 2 (other):**
```

File: content/web/development-tools-bundle/c8y-cli.md [hunk 2]
```diff
@@ -51,6 +51,57 @@ $ c8ycli new cockpit cockpit
...
 > - `c8ycli new my-cockpit cockpit -a @c8y/apps@next` will scaffold an app with the latest beta release.
 
+The `c8ycli new` command can also be provided on its own without the `[name]` and `[template]` options. In this case a few steps will have to be completed via the interface before the app is scaffolded:
+
+**Step 1:**
+```
+? Enter the name of the project: (my-application)
+```
+
+The first step will ask for project name. If no project name is entered, the default value `my-application` will be used instead.
```
Comment (BeateRixen) [inline content/web/development-tools-bundle/c8y-cli.md:61]: for project name > for the project name

**Review by rssag — DISMISSED** [review]

File: content/web/development-tools-bundle/c8y-cli.md [hunk 3]
```diff
@@ -51,6 +51,57 @@ $ c8ycli new cockpit cockpit
...
+> 1010.0.X (latest)
+> 1011.X.0 (next)
+> 1011.0.X
+> 1009.0.X
+> 1007.0.X
+> 1006.0.X
+> other
+```
+
+In the second step, the base scaffolding version must be selected. The interface will provide the last 5 major GA releases, the `latest` and `next` versions as options. Additionally a version can be manually entered by selecting the `other` option.
```
Comment (MWindel) [inline content/web/development-tools-bundle/c8y-cli.md:77]: the `latest` and `next` versions as options > does this refer to the latest and next version of the GA release? 
If so: "with the `latest` and `next` versions as options"

Comment (eniosultan) [reply content/web/development-tools-bundle/c8y-cli.md:77]: @MWindel  no, usually latest refers to the "latest stable version" while next refers to "upcoming version"

Comment (MWindel) [reply content/web/development-tools-bundle/c8y-cli.md:77]: @eniosultan thank you for clarifying. 
 
then it should either be "with the latest and next versions as options" or "the latest and next versions as options are options".

Comment (eniosultan) [reply content/web/development-tools-bundle/c8y-cli.md]: Changed the wording

File: content/web/development-tools-bundle/c8y-cli.md (hunk 2 above)
Comment (eniosultan) [reply content/web/development-tools-bundle/c8y-cli.md]: fixed

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@janhommes @DawidSAG Can one of you please approve the PR fro technical perspective?

**Review by janhommes — APPROVED** [review]

**Pull request merged**